src/plugins/tracking/_activityTrackers.js

```diff
@@ -7,13 +7,9 @@ import { updateTrackId, newSessionId } from './util/idUtils';
 const parseUrl = url.parse;
 
 /*
- * This module exports specific tracking functions that consume the `response`
+ * This module exports specific tracking functions that consume the `request`
  * object and any additional arguments that are relevant to generating the
  * tracking record.
- *
- * Note that some trackers use cookies to maintain state across sessions - these
- * functions are not guarantees to be pure because they may use the
- * `response.state` interface to set long-lived cookies
```

Comment:
I know this is based off of `request.response.state` now, but should we still mention here that we are still setting those cookies?